### PR TITLE
Doc for troubleshooting queues

### PR DESCRIPTION
**Pull request by alculquicondor**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:

Completing the troubleshooting puzzle.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: site/content/en/docs/tasks/troubleshooting/troubleshooting_queues.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+---
+title: "Troubleshooting Queues"
+date: 2024-03-21
+weight: 2
+description: >
+  Troubleshooting the status of a ClusterQueue
```
Comment (tenzen-y) [inline site/content/en/docs/tasks/troubleshooting/troubleshooting_queues.md:6]:
```suggestion
  Troubleshooting the status of a ClusterQueue and a LocalQueue
```

👀

**Review by tenzen-y — COMMENTED** [review]
Just a nit.

File: site/content/en/docs/tasks/troubleshooting/troubleshooting_queues.md (hunk 1 above)
Comment (alculquicondor) [reply site/content/en/docs/tasks/troubleshooting/troubleshooting_queues.md]: fixed :)

**Review by tenzen-y — COMMENTED** [review]
/lgtm

**Pull request merged**